src/GraphQL/Types/ResolveFieldContext.cs

```diff
@@ -79,32 +79,47 @@ public ResolveFieldContext(ResolveFieldContext context)
 
         public TType GetArgument<TType>(string name, TType defaultValue = default)
```

Comment:
previous implementation of this method resulted in `defaultValue` parameter boxing